<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>einLengthMeasure</class>
 <widget class="QDockWidget" name="einLengthMeasure">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>948</width>
    <height>593</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DockWidget</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <widget class="QLabel" name="img_continue">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="save">
    <property name="geometry">
     <rect>
      <x>776</x>
      <y>540</y>
      <width>80</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>save</string>
    </property>
   </widget>
   <widget class="QPushButton" name="grab">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>540</y>
      <width>80</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>grab</string>
    </property>
   </widget>
   <widget class="QPushButton" name="measure">
    <property name="geometry">
     <rect>
      <x>862</x>
      <y>540</y>
      <width>80</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>measure</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="forCal">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>510</y>
      <width>124</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>forCal</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="imgName">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>510</y>
      <width>113</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="length">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>480</y>
      <width>113</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="img_continue_2">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>480</y>
      <width>90</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>length</string>
    </property>
   </widget>
   <widget class="QLabel" name="img_continue_3">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>510</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>name</string>
    </property>
   </widget>
   <widget class="QPushButton" name="func">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>480</y>
      <width>80</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>func</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
